Fedlet/Saml2/IdentityProvider.cs

```diff
@@ -87,7 +88,7 @@ public IdentityProvider(XmlDocument metadata, XmlDocument extendedMetadata, Saml
 
 				// Load now since a) it doesn't change and b) its a 
 				// performance dog on Win 2003 64-bit.
-				byte[] byteArray = Encoding.UTF8.GetBytes(EncodedSigningCertificate);
+				byte[] byteArray = Convert.FromBase64String(EncodedSigningCertificate);

```

Comment:
This looks like a breaking change. I understand why you are making the change, just not sure if it's enough. Let's think about what else we have to update (new version number? anything else to put in?).

Reply:
This is the only place where X509Certificate2 is being created from string. All other functionality seems to work fine both on Linux (Ubuntu) and Windows, tested on several identity providers we have access to, .net core 3.1 and .net 5.0.
So I’m not sure what else should be done here aside from bumping version number.

Reply:
This is the only place where X509Certificate2 is being created from string. All other functionality seems to work fine both on Linux (Ubuntu) and Windows, tested on several identity providers we have access to, .net core 3.1 and .net 5.0.
So I’m not sure what else should be done here aside from bumping version number.

Reply:
@romanpolunin Do you agree? We can have a call regarding this if you want more details but we do not expect that the change is breaking.

Reply:
More over standard tells that certificate must be a Base64 encoded. 

> ...X509Certificate element, which contains a base64-encoded [X509V3] certificate...